### ISSUE TITLE
The atom feed shows the unformatted markup

### ISSUE THREAD

**Issue by ArtskydJ** [saibotsivad/saibotsivad.github.io#11]
http://saibotsivad.com/feed.xml

```xml
...
<description>
{% raw %} In my current work with [From Now On](http://from-now-on.com/) I inherited a CMS website that was built with old technology, created in a rush, and had a lot of [fundamental](https://medium.com/javascript-scene/how-to-fix-the-es6-class-keyword-2d42bb3f4caf) issues.
...
```

This causes a feed reader to actually show the text `[From Now On](http://from-now-on.com/)` instead of having a link...

**Comment by saibotsivad** [comment]
That's unfortunate, the double curly braces were throwing off Jekyll in the template. I'll look into it to see if I can find a solution. Thanks for letting me know!

On July 14, 2017 7:03:43 PM CDT, Joseph Dykstra <notifications@github.com> wrote:
>http://saibotsivad.com/feed.xml
>
>```xml
>...
><description>
>{% raw %} In my current work with [From Now
>On](http://from-now-on.com/) I inherited a CMS website that was built
>with old technology, created in a rush, and had a lot of
>[fundamental](https://medium.com/javascript-scene/how-to-fix-the-es6-class-keyword-2d42bb3f4caf)
>issues.
>...
>```
>
>This causes a feed reader to actually show the text `[From Now
>On](http://from-now-on.com/)` instead of having a link...
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/saibotsivad/saibotsivad.github.io/issues/11

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by saibotsivad** [comment]
I got the feed rendering the markdown to HTML :tada:
